common/types.go

```diff
@@ -448,6 +450,70 @@ func (e *VersionedExecutionPayload) NumTx() int {
 	return 0
 }
 
+func encodeTransactions(txs []*types.Transaction) [][]byte {
```

Comment:
this file should probably be its own PR with a full set of unit tests. types are a lot of boiler plate but should be really testable